### DISCUSSION TITLE
cefsharp is not support   audio/mp4; codecs="mp4a.40.2"???

### DISCUSSION THREAD

**Discussion by whiskets** [cefsharp/CefSharp#4557] [Q&A]
l try to use cefsharp display a <iframe> embeds about spotify music

but if l tap the play button then will show a error

**"Uncaught NotSupportedError: Failed to execute 'addSourceBuffer' on 'MediaSource': The type provided ('audio/mp4; codecs="mp4a.40.2"') is unsupported.", source: https://open.spotifycdn.com/cdn/build/embed/vendor~embed.4d35432e.js** 

so is that my cefsetting porblem or have some method to support this codecs

**Comment by amaitland** [comment]
`CefSharp` only supports `free/open` codecs. Not supported due to licensing restrictions.

https://github.com/cefsharp/CefSharp/wiki/General-Usage#multimedia-audiovideo
